menpo/shape/mesh/base.py

```diff
@@ -227,6 +229,77 @@ def init_from_depth_image(cls, depth_image):
     def __str__(self):
         return "{}, n_tris: {}".format(PointCloud.__str__(self), self.n_tris)
 
+    def __add__(self, other):
```

Comment:
This would be a huge change to the menpo API and should definitely be put up as a different PR where we can discuss if we want this. We had always resisted this change specifically to stop people from confusing our objects with numpy arrays - but we can re-open the discussion

Reply:
Ok,  I don't have hard feelings for this change. I dont think that I used it many times in my code but we can discuss if it worths to have them.